.ci/check

```diff
@@ -53,3 +53,14 @@ golint -set_exit_status ${LINT_FOLDERS}
 
 # Execute lint checks on helm chart.
 helm lint "${HELM_CHART_PATH}"
+
+echo "Checking Go version"
+GOVERSION=$(go list -f {{.GoVersion}} -m)
+if [[ $GOVERSION =~ ^[0-9]+\.[0-9]+\.0$ ]]; then
+  # Go version is valid, adheres to x.y.0 version
+  exit 0
+else
+  echo "Go version is invalid, please adhere to x.y.0 version"
+  echo "See https://github.com/gardener/etcd-backup-restore/pull/801"
+  exit 1
+fi
```

Comment:
Can you also add `echo "All checks successful"` at end as was done in druid PR.

Reply:
Thanks for the suggestion @anveshreddy18!